docs/deploy-required-infrastructure.md

```diff
@@ -294,6 +294,133 @@ kubectl apply -k /opt/genestack/kustomize/ovn
 After running the setup, nodes will have the label `ovn.openstack.org/configured` with a date stamp when it was configured.
 If there's ever a need to reconfigure a node, simply remove the label and the DaemonSet will take care of it automatically.
 
+## Deploy PostgreSQL
+
+### Create Secrets
+
+```shell
```

Comment:
```suggestion
``` shell
```

Reply:
I can change this if we want to conform to a standard for genestack, but I think the actual spec is ```\<identifier\> with no space.
- https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting
- https://www.markdownguide.org/extended-syntax/#syntax-highlighting